pkg/services/contexthandler/authproxy/authproxy.go

```diff
@@ -318,18 +328,20 @@ func (auth *AuthProxy) GetSignedInUser(userID int64) (*models.SignedInUser, erro
 
 // Remember user in cache
 func (auth *AuthProxy) Remember(id int64) error {
-	key := auth.getKey()
+	key, err := auth.getKey()
+	if err != nil {
+		return err
+	}
 
 	// Check if user already in cache
-	userID, _ := auth.remoteCache.Get(key)
-	if userID != nil {
+	userID, err := auth.remoteCache.Get(key)
```

Comment:
Shouldn't we return if `err != nil`?

Reply:
@wbrowne In this particular case, we want to return if the user is already in the cache, i.e. if no error is returned and `userID` is non-nil. The old code didn't take into account the error.

Reply:
But in the case where there is an error?

Reply:
@wbrowne As I think the original logic is, it's fine to ignore the cache if an error occurs looking it up.